### PR TITLE
Add Crack & Stack project

### PR DESCRIPTION
**Pull request by vimtor**
**NOTE**

#### Please enable "Allow edits by maintainers" while putting up the PR.

---

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
4. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
5. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
6. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Name (to be shown on DefiLlama):
Crack & Stack

##### Twitter Link:
https://x.com/crackandstack/

##### List of audit links if any:
Not an audit, but we are an official Taiko partner:
https://taiko.xyz/ecosystem?search=crack+%26+stack

##### Website Link:
https://crackandstack.com/

##### Logo (High resolution, will be shown with rounded borders):
https://crackandstack.com/profile.png

##### Current TVL:
4.09 ETH

##### Chain:
Taiko

##### Short Description (to be shown on DefiLlama):
Crack & Stack is the leading multiplayer Web3 game on Taiko. Enter the mines, collect ETH and try to escape with your bounty.

##### Category (full list at https://defillama.com/categories) *Please choose only one:
Gaming

##### methodology (what is being counted as tvl, how is tvl being calculated):
Users can purchase an ERC-1155 token that we call lanterns ([address](https://taikoscan.io/address/0x7ddB8A975778a434dE03dd666F11Ce962DCdD290)). These lanterns are used to enter paid mines (game rooms). In these mines, users find diamonds that they can click to accumulate ETH. If they survive for long enough, we send them their ETH bounty using the contracts in this PR.

When a lantern is purchased, we need to back that value since users will win part of it when playing the game. Lanterns can also be acquired from other sources like referrals or airdrops. When they are minted, we send that value to the contracts in this PR.

In summary, we sum the value that our reward-sending smart contracts hold.

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/crackandstack exports TVL: 
        
 
 
    taiko                     11.11 k
    
    total                    11.11 k

**Comment by g1nt0ki** [comment]
@vimtor thanks for the PR
